### ISSUE TITLE
Windows permission issue?

### ISSUE THREAD

**Issue by cloudbluedigital** [atom/atom#17477]
### Description

Whenever Atom with a project is opened and I try to use npm or git from command line, I get a permission issue that a file can't be removed etc. so if for example I do git checkout unless I close Atom it will say that it cannot switch branches because modified files give permission denied. Seems like it's only an issue when Atom is open

### Steps to Reproduce

1. Launch atom and add project folder
2. Make a change to any file which is a git repository
3. go to cmd.exe/powershell.exe and type git checkout 'different_branch' or run npm run prod (anything that modifies files really)

**Expected behavior:** [What you expect to happen]

Git succeding and npm builds not failing

**Actual behavior:** [What actually happens]

Both are failing due to permission denied issue?

**Reproduces how often:** [What percentage of the time does it reproduce?]

every time

### Versions

You can get this information from copy and pasting the output of `atom --version` and `apm --version` from the command line. Also, please include the OS and what version of the OS you're running.

```
C:\Coding\website_api [develop ≡]
λ  atom --version

Atom    : 1.27.2
Electron: 1.7.15
Chrome  : 58.0.3029.110
Node    : 7.9.0
C:\Coding\website_api [develop ≡ +0 ~1 -0 !]
λ  apm --version
apm  1.19.0
npm  3.10.10
node 6.9.5 x64
atom 1.27.2
python 3.6.5
git 2.17.1.windows.2
visual studio
```

Windows 10 with latest 1803 build

**Comment by rsese** [comment]
Thanks for the report - I think this might be a duplicate of https://github.com/atom/atom/issues/17013.  Assuming it doesn't exist, can you create a `recovery` directory in your Atom home directory and see if that fixes things for you?

**Comment by cloudbluedigital** [comment]
I still have an issue
```
sh.exe C:\Users\arthurg\AppData\Local\Atlassian\SourceTree\gitflow_local\gitflow\git-flow release finish -p -f C:\Users\arthurg\AppData\Local\Temp\wdk4jxql.o0t 0.50.0-beta
Your branch is up to date with 'origin/master'.
Switched to branch 'master'
error: unable to create file pages/information/privacy-policy.vue: Permission denied
Fatal: There were merge conflicts.
Fatal: Working tree contains unstaged changes. Aborting.
Completed with errors, see above.

```

**Comment by rsese** [comment]
> I still have an issue

Just to confirm, did you not have a `recovery` directory in your Atom home folder? Then you created one and restarted Atom but you're still getting the same error?

Also, you can still reproduce with the original repro steps you shared (i.e. checking out a different branch)?

**Comment by cloudbluedigital** [comment]
yes @rsese exactly that. Folder was not there so I created it and restarted atom.

Yes the issue above shows that after a switch to master branch there is permission denied on one of the files which doesn't exist

**Comment by rsese** [comment]
Hmm, can you try the 1.28 beta and see if the behavior is the same there?

**Comment by Matty666** [comment]
I've just updated to `v1.28 x64` and am getting the same issue. If I open a folder, and use git on the command line to change branch i get heaps of permission denied errors. It fails and leaves the working tree in a funny state. trying to checkout or reset also fails. Close atom and everything works. I checked and I do have a recovery folder in `%USERPROFILE%\.atom` but it's empty.

**Comment by chamra** [comment]
I have the same  issue. I'm using v1.28.0

**Comment by rsese** [comment]
I just tested on Windows 10 with Atom 1.28.0 but wasn't able to reproduce (I tried switching branches from the command line).  I do have a `recovery` directory in my Atom home directory but I went ahead and deleted it and restarted Atom to test again, but the folder is recreated (I'm assuming as expected since https://github.com/atom/atom/issues/17013 was fixed).

@Matty666 @chamra - did you also have trouble in previous versions of Atom or did it just start with 1.28.0 for you?
